lib/libutee/include/tee_api.h

```diff
@@ -114,7 +114,7 @@ bool TEE_MaskCancellation(void);
 TEE_Result TEE_CheckMemoryAccessRights(uint32_t accessFlags, void *buffer,
 				       uint32_t size);
 
-void TEE_SetInstanceData(void *instanceData);
+void TEE_SetInstanceData(const void *instanceData);
```

Comment:
@etienne-lms OK you convinced me. Because of `void` a cast is needed anyway.

\<rant mode="on"\> I find it a bit silly that the spec is not giving the proper C prototypes. It says "[The const C keyword] is omitted in the specification of the functions to avoid mixing the formal annotations and a less expressive C keyword.". And I say "BS, annotate if you wish but write proper prototypes, you're defining an API!"\<rant mode="off"\>